object_manager.go

```diff
@@ -157,7 +160,7 @@ func BuildNetworkFromRef(ref string) *Network {
 	}
 }
 
-func (objMgr *ObjectManager) GetNetwork(netview string, cidr string, ea EA) (*Network, error) {
+func (objMgr *ObjectManager) GetNetwork(netview string, cidr string, eaSearch EASearch) (*Network, error) {
```

Comment:
This changes the function signature thereby breaking current docker ipam plugin. I am not sure if any other project uses it.